examples/windows/kubernetes-hybrid.containerd.json

```diff
@@ -0,0 +1,55 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "orchestratorRelease": "1.14",
+      "kubernetesConfig": {
+        "networkPlugin": "kubenet",
+        "containerRuntime": "containerd",
+        "windowsContainerdURL": "https://acsenginedev.blob.core.windows.net/cri-containerd/windows-cri-containerd-05072019.zip",
+        "windowsSdnPluginURL": "https://acsenginedev.blob.core.windows.net/cri-containerd/windows-cni-containerd-05072019.zip"

```

Comment:
@marosset - what do you think - should these be in `kubernetesConfig`, `agentPoolProfile` or `windowsProfile` ?